viewing_party/party.py

```diff
@@ -1,23 +1,248 @@
+
+# -----------------------------------------
 # ------------- WAVE 1 --------------------
+# -----------------------------------------
 
 def create_movie(title, genre, rating):
-    pass
+    #create a dictionary with the keys 'title', 'genre', 'rating', and the values the values passed in
+    if title and genre and rating:
+        movie_dict = {}
+        movie_dict['title'] = title
+        movie_dict['genre'] = genre
+        movie_dict['rating'] = rating
+        return movie_dict
+
+def add_to_watched(user_data, movie):
+    #user_data is a dictionary with watched as one of the keys.

```

Comment:
I like these comments! Consider converting them to docstring format with python multiline comments: ` ``` `
Not only do they help with formatting (not as many #s) but IDEs have support for displaying them as well as automatically generating documentation.